lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -132,6 +134,41 @@ export async function initPlatform({
     );
   }
 
+  if (!gitAuthor && username) {
```

Comment:
This PR still targets the `main` branch! I think you want the author to target the `v38` branch?

Also if this is a breaking change it should have the `!` character in the PR title, in the right place. So that Semantic Release groups it under the breaking changes section of the final 38.0.0 changelog.